incubator/hnc/README.md

```diff
@@ -53,9 +58,8 @@ objects you can't afford to lose (e.g. that aren't stored in a Git repository).
 All HNC issues are assigned to an HNC milestone. So far, the following
 milestones are defined or planned:
 
-* v1.0 - Likely late Q2 or early Q3 2021: HNC recommended for production use
-* [v0.9](https://github.com/kubernetes-sigs/multi-tenancy/milestone/21): move
-  HNC to its own repo; continued stability improvements.
+* ***Post v0.8: please visit the [new
```

Comment:
(Comment on line 35-36 `Getting started and learning more` section - "The latest version of HNC is v0.8.0")
We may mention this is the latest version in this repo and see the latest HNC in the new repo there too.

Reply:
I _think_ I've fixed this - lgty?